slither/tools/read_storage/read_storage.py

```diff
@@ -208,6 +266,78 @@ def get_target_variables(self, **kwargs) -> None:
             if slot_info:
                 self._slot_info[f"{contract.name}.{var.name}"] = slot_info
 
+    def find_constant_slot_storage_type(
+        self, var: StateVariable
+    ) -> Tuple[Optional[str], Optional[int]]:
+        """
+        Given a constant bytes32 StateVariable, tries to determine which variable type is stored there, using the
+        heuristic that if a function reads from the slot and returns a value, it probably stores that type of value.
+        Also uses the StorageSlot library as a heuristic when a function has no return but uses the library's getters.
+        Args:
+            var (StateVariable): The constant bytes32 storage slot.
+
+        Returns:
+            type (str): The type of value stored in the slot.
+            size (int): The type's size in bits.
+        """
+        assert var.is_constant and var.type == ElementaryType("bytes32")
+        storage_type = None
+        size = None
+        funcs = []
+        for c in self.contracts:
+            c_funcs = c.get_functions_reading_from_variable(var)
+            c_funcs.extend(
+                f
+                for f in c.functions
+                if any(str(v.expression) == str(var.expression) for v in f.variables)
+            )
+            c_funcs = list(set(c_funcs))
+            funcs.extend(c_funcs)
+        fallback = [f for f in var.contract.functions if f.is_fallback]
+        funcs += fallback
+        for func in funcs:
+            rets = func.return_type if func.return_type is not None else []
+            for ret in rets:
+                size, _ = ret.storage_size
+                if size <= 32:
+                    return str(ret), size * 8
```

Comment:
```suggestion
        for func in funcs:
            if func.return_type:
              for ret in rets:
                  size, _ = ret.storage_size
                  if size <= 32:
                      return str(ret), size * 8
```
You probably can't accept this directly due to the whitespace btw

Reply:
I believe I wrote this the way I did in order to avoid a pylint `too-many-branches` complaint, since I'm right there at the maximum of 12 branches. Adding `if func.return_type` puts it at 13.

Should I leave it this way, or disable `too-many-branches` for this function?